src/modules/Dashboard/pages/Applet/Applet.styles.tsx

```diff
@@ -0,0 +1,8 @@
+import { styled } from '@mui/material';
+
+import { variables } from 'shared/styles';
+
+export const StyledAppletLogo = styled('img')({
+  width: '48px',
```

Comment:
Please use rem instead of px. You can define a rem value by dividing the px value by 10.
```suggestion
  width: '4.8rem',
```